### ISSUE TITLE
Need more heredity in GENO

### ISSUE THREAD

**Issue by mbrush** [monarch-initiative/GENO-ontology#2]
**Transferred from:** [NIF-11673](https://support.crbs.ucsd.edu/browse/NIF-11673)
**Original Reporter:** Nicole Washington

---

We need to expand the genetic heritability subclasses to include:

X- Y- Z- linked (allosomal)
autosomal
unknown genetic heritability

these should probably be linked to the phenotypic dominance terms. these will get used when the genetic basis of inheritance is known, but the dominance is unknown.

**Comment by pnrobinson** [comment]
Final comment: Inheritance also is not part of the actual genotype, and it is debatable whether it belongs in GENO. What actually is the mission statement of GENO?
-Peter

Dr. med. Peter N. Robinson, MSc.
Professor of Medical Genomics
Professor in the Bioinformatics Division of the Department of Mathematics and Computer Science of the Freie Universität Berlin
Institut für Medizinische Genetik und Humangenetik
Charité - Universitätsmedizin Berlin
Augustenburger Platz 1
13353 Berlin
Germany
+4930 450566006
Mobile: 0160 93769872
peter.robinson@charite.de
http://compbio.charite.de
http://www.human-phenotype-ontology.org
Introduction to Bio-Ontologies: http://www.crcpress.com/product/isbn/9781439836651
I have learned from my mistakes, and I am sure I can repeat them exactly
ORCID ID:http://orcid.org/0000-0002-0736-9199
Scopus Author ID 7403719646
Appointment request: http://doodle.com/pnrobinson

---

Von: mbrush [notifications@github.com]
Gesendet: Freitag, 16. Januar 2015 01:44
An: monarch-initiative/GENO-ontology
Betreff: [GENO-ontology] Need more heredity in GENO (#2)

Transferred from: NIF-11673https://support.crbs.ucsd.edu/browse/NIF-11673
Original Reporter: Nicole Washington

---

We need to expand the genetic heritability subclasses to include:

X- Y- Z- linked (allosomal)
autosomal
unknown genetic heritability

these should probably be linked to the phenotypic dominance terms. these will get used when the genetic basis of inheritance is known, but the dominance is unknown.

**Comment by mbrush** [comment]
@pnrobinson  Regarding the mission and scope of GENO, the readme for this repo is currently the main public documentation on this.  I updated it today to give a bit more detail, and will continue to improve this and add content to the wiki.  While the primary charge of GENO is to represent the different levels of genetic variation represented in genotypes, it also holds concepts related to genetic variation and genotype-to phenotype associations.  In this area the boundaries of its scope are fuzzy. More detail in the readme and to come on the wiki.

**Comment by nlwashington** [comment]
here is a set of inheritance used by OMIA (which covers >200 species):

Autosomal
Autosomal Co-Dominant  =  GENO:0000143
Autosomal Dominant  =  GENO:0000147
Autosomal Incompletely Dominant  =  GENO:0000259
Autosomal Recessive  =  GENO:0000150
Autosomal Recessive Lethal
Autosomal Recessive, Semi-Lethal
Autosomal Semi-Dominant  =  GENO:0000145
Autosomal dominant with variable expressivity
Dominant  =  GENO:0000144
Maternal
Multifactorial
Probably Autosomal Recessive
Recessive Embryonic Lethal
Sex limited
Sex-linked  = GENO:0000146  (this is allosomal dominant, which isn't quite right)
X-linked
X-linked dominant
X-linked recessive
Y-linked
Z-linked
Z-linked incompletely dominant
Z-linked recessive

only a small fraction of these map to geno, as noted.  can you make new terms for these, where appropriate?  relevant for https://github.com/monarch-initiative/dipper/issues/9

**Comment by mbrush** [comment]
I reviewed several tickets (#2, #6, #18) relating to the issue of defining a hierarchy of phenotypic inheritance classes in GENO, to describe the mode of inheritance of traits/phenotypes/diseases.

As per Peter and Melissa's comments in #6, we will define classes generically for all taxa, and make equivalence or subsumption links where appropriate to terms in existing vocabularies such as those in the 'Mode of Inheritance' hierarchy in the HPO. 

The inheritance classes will live in GENO, unless there is a more appropriate ontology I am not aware of.  I found scant mention of something called the 'Phenotypic Manifestation' ontology ( [link1](http://www.ebi.ac.uk/ontology-lookup/browse.do?ontName=PM), [link2](http://obo.cvs.sourceforge.net/obo/obo/ontology/phenotype/context.obo) ) - but doesn't seem like this is really a thing anymore.

So, ontologically speaking, mode of inheritance terms describe the pattern in which a particular genetic trait or disorder is passed from one generation to the next.  Different modes of inheritance are distinguished along three main axes of classification:
1. the **number of contributing loci** (e.g. monogenic vs polygenic inheritance)
2. the **chromosome type** where a contributing locus resides (e.g. autosomal vs sex-linked inheritance)
3. the **pattern of dominance** associated with the causative allele(s) for a trait (e.g. dominant vs recessive inheritance)

This final axis of dominance patterns applies for monogenic inheritance, and is most complex and nuanced. It is based on how the genetic interactions between alleles at the causative locus determine the pattern of inheritance of a specific phenotype from one generation to the next. 

Two proposals for inheritance hierarchies are shown below, which are informed by the HPO, FBcv, and OMIA requirements as listed above- such that our model will satisfy our immediate requirements to ingest and operate over this data.

**Proposal 1**:  _Creates classes based on multiple axes in combination for a deeper hierarchy of more specific terms_

```
multifactorial inheritance
    digenic inheritance
    oligogenic inheritance
    polygenic inheritance

monogenic inheritance (aka single-locus inheritance)
    autosomal inheritance
        autosomal complete dominant inheritance
        autosomal incomplete dominant inheritance (aka semi-dominant, intermediate dominant)
        autosomal co-dominant inheritance
        autosomal recessive inheritance

    allosomal inheritance (aka gonosomal)
        X-linked inheritance
            X-linked complete dominant inheritance
            X-linked co-dominant inheritance
            X-linked recessive inheritance
        Y-linked inheritance (aka holandric inheritance)
        Z-linked inheritance
            Z-linked dominant inheritance
            Z-linked incomplete dominant inheritance
            Z-linked recessive inheritance
        W-linked inheritance
    mitochondrial inheritance

unknown inheritance (required of data where mode of inheritance is not known)
```

One issue here is that the order of how the axes are applied results in like terms not being grouped together.  For example, co-dominant inheritance classes are split between across allosomal and autosomal branches. New grouping classes such as 'co-dominant inheritance' and (asserted or inferred) multiple inheritance would be required if we require these groupings to be made.

**Proposal 2**: _An alternate approach creates classes only based on one axis - i.e. no combinatorial class creation._  This results in a simpler model of fewer classes, with flexibility for combining terms in annotations to build a complete picture of inheritance patterns. For example,  a phenotype might be annotated with 'monogenic inheritance', 'X-linked inheritance', and 'co-dominant inheritance',  instead of just the single more precise term 'monogenic X-linked co-dominant inheritance' term.

```
multifactorial inheritance
    digenic inheritance
    oligogenic inheriitance
    polygenic inheritance

monogenic inheritance (aka single-locus inheritance)

autosomal inheritance

allosomal inheritance (aka gonosomal)
    X-linked inheritance
    Y-linked inheritance (aka holandric inheritance)
    Z-linked inheritance
    W-linked inheritance

dominant inheritance
    complete dominant inheritance
    incomplete dominant inheritance (aka semi-dominant, intermediate dominant)
    co-dominant inheritance

recessive inheritance
    recessive lethal inheritance
    recessive semi-lethal inheritance

mitochondrial inheritance

unknown inheritance
```

Feedback welcome. Of note, we will eventually have to commit to the ontological nature of these mode of inheritance classes. Do we treat them as **processes** (i.e. subtypes of a generic 'phenotype inheritance' process)?  As **qualities of an inheritance process**?  Or as **qualities or dispositions of traits** themselves?  I created a separate ticket for this (#22) .

**Comment by mbrush** [comment]
Also of note, some of the terms requested for OMIA seem to conflate inheritance and other aspects of phenotype manifestation. For example 'Autosomal dominant with variable expressivity'.  Should this be captured with annotations to two terms, 'complete dominant autosomal inheritance' and 'variable expressivity'? I prefer this over cramming the notion of expressivity into the inheritance hierarchy.  Same for 'Autosomal recessive lethal inheritance' and 'Autosomal recessive semi-lethal inheritance' . . . not sure I am comfortable basing inheritance classes on the notion of lethality of the phenotype?

A full list of questionable terms: 
- Autosomal Recessive Lethal
- Autosomal Recessive, Semi-Lethal
- Autosomal dominant with variable expressivity
- Maternal
- Probably Autosomal Recessive
- Recessive Embryonic Lethal

**Comment by pnrobinson** [comment]
I agree these terms should be "de-bundled"
-Peter

Dr. med. Peter N. Robinson, MSc.
Professor of Medical Genomics
Professor in the Bioinformatics Division of the Department of Mathematics and Computer Science of the Freie Universität Berlin
Institut für Medizinische Genetik und Humangenetik
Charité - Universitätsmedizin Berlin
Augustenburger Platz 1
13353 Berlin
Germany
+4930 450566006
Mobile: 0160 93769872
peter.robinson@charite.de
http://compbio.charite.de
http://www.human-phenotype-ontology.org
Introduction to Bio-Ontologies: http://www.crcpress.com/product/isbn/9781439836651
I have learned from my mistakes, and I am sure I can repeat them exactly
ORCID ID:http://orcid.org/0000-0002-0736-9199
Scopus Author ID 7403719646
Appointment request: http://doodle.com/pnrobinson

---

Von: mbrush [notifications@github.com]
Gesendet: Montag, 14. September 2015 22:20
An: monarch-initiative/GENO-ontology
Cc: Robinson, Peter
Betreff: Re: [GENO-ontology] Need more heredity in GENO (#2)

Also of note, some of the terms requested for OMIA seem to conflate inheritance and other aspects of phenotype manifestation. For example 'Autosomal dominant with variable expressivity'. Should this be captured with annotations to two terms, 'complete dominant autosomal inheritance' and 'variable expressivity'? I prefer this over cramming thee notion of expressivity into the inheritance hierarchy. Same for 'Autosomal recessive lethal inheritance' and 'Autosomal recessive semi-lethal inheritance' . . . not sure I am comfortable basing inheritance classes on the notion of lethality of the phenotype?

A full list of questionable terms:
-   Autosomal Recessive Lethal
-   Autosomal Recessive, Semi-Lethal
-   Autosomal dominant with variable expressivity
-   Maternal
-   Probably Autosomal Recessive
-   Recessive Embryonic Lethal

**Comment by mellybelly** [comment]
I agree, don't like lethality being bundled. This one especially is a phenotype. We need to have a better way of representing lethality at different stages, but this is a separate issue. 

We do need an expressivity term, but I think this would be a PATO term.  

I think these terms can live in GENO and then they can get mireoted into the respective phenotype ontologies as needed.

I prefer the first proposal with logical definitions rather than creating multiple annotations to each atomic term.

**Comment by mellybelly** [comment]
Also keep the unknown, i know its not "ontologically correct", but we need to easily parse out the assertions of unknown heritability rather than annotating up.

**Comment by mbrush** [comment]
Translating Melissa's feedback from our discussion today:
- prefers Proposal 1 of classifying along multiple dimensions and creating defined classes to get a polyhierarchy with inferred multiple inheritance to the extent it is found useful. So we'd need to create logical definition patterns to get multiple inheritance.
- agrees we should separate lethality from our inheritance classes.  Expressivity is a less clear, as it can be framed as a quality of an inheritance process, and thus be used to classify types of inheritance.
- with respect to the ontological nature of inheritance types, see comments in #22.

@cmungall can you weigh in on this ticket as well.

**Comment by beckyjackson** [comment]
Hi @mbrush - I'm curious to know if there has been any more work done on including the X/Y linked inheritance classes in GENO? We are now using GENO in DO and would like to use these specific classes. Thank you!

**Comment by mbrush** [comment]
Hi @rctauber.  A few new inheritance-related classes have made their way into GENO, but nothing approaching the full proposal specced out above.  We could probably push forward on this if there is a dependency for your work with DO. Its been something we've wanted to do for a while anyway, and would have utility in other efforts as well. As noted above, one dependency for implementation is coordinating with HPO where some of these terms also live.
 
Do you have any specific requirements for classes or their organization in the ontology?  Would the approach outlined in Proposal 1 above meet all your needs? If not, what else would be required?

**Comment by beckyjackson** [comment]
Proposal 1 would absolutely work for us, and having those specific children would also be fantastic. We import the 'condition inheritance' node and have been using the 'allosomal [dominant/recessive] inheritance' class for X-linked inheritance, but we need to differentiate between X- and Y-linked. So having the X-linked and Y-linked children for allosomal inheritance would be extremely helpful.

Proposal 2 would be a bit more difficult to work with, as our design pattern stands. But if that's the direction you decide to go, we can make that work :)

**Comment by pnrobinson** [comment]
I think that heredity is not really in scope of a genotype ontology. While perhaps one can say that a dominant genotype can be inherited from a parent, a homozygous genotype cannot (etc).  Also, the phenotypic consequences of a genotype at a given locus may depend on genetic background, and thus whether or not a trait is considered autosomal dominant may depend on the situation.

**Comment by mellybelly** [comment]
We have discussed including mode of inheritance in GENO for a long time and there have been many requests for it. I don't think there is enough content to warrant a different ontology, and imho it would make sense to most users to include it in GENO since they are often recording genotype information alongside inheritance. I'm also keen to have this information in a species-neutral location, and GENO aims to have pan-species utility.

**Comment by beckyjackson** [comment]
Hi again @mbrush - I just wanted to check on the status of this. Have you reached a decision for these new terms?

**Comment by mbrush** [comment]
Hi @rctauber. Sorry for dropping the ball here - I haven't had a chance to sync up with @mellybelly and @pnrobinson on this yet.  But I think there is general agreement that Proposal 1 above is preferred - which  creates "inheritance pattern" (aka "mode of inheritance") classes based on multiple axes in combination, for a deeper hierarchy of very specific terms.
 
I have fleshed out the proposal with a more complete hierarchy and term definitions in the gdoc [here](https://docs.google.com/spreadsheets/d/1i29O9c0E4ge3Ztv9pHWaLT4hQvYEINNce4ORoDuY03c/edit#gid=0).  I aimed to be as consistent as possible with the HP terms/definitions, where provided.

@mellybelly @pnrobinson @cmungall  @rctauber please review/propose changes so I can implement and get DO the terms they need.

**Comment by pnrobinson** [comment]
I do not think this is a good idea. Inheritance and genotypes are not intimately connected.
This is completely duplicative of the corresponding hierarchy in the HPO. The terms are also human centric. It is also a mistake to call a genotype polygenic. This is a mistake in modeling.
I do not see the need for an additional hierarchy of terms in Geno. Is there model organism data for which we want to annotate inheritance? 
Can we please discuss this via zoom before you go ahead?

**Comment by mellybelly** [comment]
There is a need to have heredity representation for all organisms, HPO is only for human. I suggest we implement this in GENO and the two ontologies can align and cross reference (or HPO could cede to a more generic place). I don't think we want to have another ontology for this purpose, though that could be an alternative if we don't feel that heredity and genotype components belong in the same ontology. I personally don't think this matters and prefer not to create more ontologies, it would be a different root, not dissimilar to HPO having age ranges as a different branch - these are not phenotypes but are helpful to have handy in that context.

**Comment by pnrobinson** [comment]
I think that for model organisms, what is needed is different, since there is always an interplay between experimentation, genetic manipulation, and phenotype. Can you point out some data that we would want to annotate in this way? Maybe I am not understanding the use case.

**Comment by lschriml** [comment]
Thank you @mbrush for moving this forward.
Our use case, is to be able to differentiate (and annotate) the mode of inheritance for OMIM phenotype terms that are aligned to DO diseases. We are extending the DO's logical definitions. There are cases where a disease is known to be inherited - X-linked, but whether that inheritance is dominant or recessive is unknown. 

Thinking about GENO in a broader context, and following on @mellybelly Melissa's comments, 
I think GENO can bring together the human, model organism, haploid, diploid and even epigenetic modes of inheritance.

I have included some specific examples with sources below. 

Cheers,
Lynn 


For GENO, can you cross-link with the **GO's set of inheritance terms:**
   This is the set used in JAX: http://www.informatics.jax.org/vocab/gene_ontology/GO:0001319
(searching on this page for : inheritance)

mitochondrion inheritance
nucleus inheritance
endoplasmic reticulum inheritance
plastid inheritance
vacuole inheritance
peroxisome inheritance
organelle inheritance
Golgi inheritance
mitochondrial DNA inheritance
regulation of Golgi inheritance

I would encourage GENO to also model the mode of inheritance for all of the **model organisms.**
     e.g.       genetics of mice: http://www.informatics.jax.org/silver/chapters/1-2.shtml

--> Evidence for the applicability of Mendel's laws to mammals — and by implication, to humans — came quickly, with a series of papers published by the French geneticist Cuénot on the inheritance of the various coat color phenotypes (Cuénot, 1902; Cuénot, 1903; Cuénot, 1905). Not only did these studies confirm the simple dominant and recessive inheritance patterns expected from "Mendelism", they also brought to light additional phenomena such as the existence of more than **two alleles at a locus, recessive lethal alleles, and epistatic interactions among unlinked genes.**


And to include haploid as well as **diploid organisms**. 
see NCBI Book:
Mendelian genetics in eukaryotic life cycles
https://www.ncbi.nlm.nih.gov/books/NBK21836/

[Which includes: ]
--> Do all these various life cycles show Mendelian genetics? The answer is that Mendelian inheritance patterns characterize any species that has meiosis as part of its life cycle, because Mendelian laws are based on the process of meiosis. All the groups of organisms mentioned, except bacteria, undergo meiosis as part of their cycles.


Have you considered including **epigenetic inheritance,**
e.g.
https://www.ncbi.nlm.nih.gov/books/NBK22029/

**Comment by pnrobinson** [comment]
I do not think it would be good to mix items such as golgi inheritance, which refer to the way that subcellular organs are transmitted to daughter cells, and inheritance terms that describe how traits or diseases are transmitted!
